tasks/src/image-segmentation/data.ts

```diff
@@ -0,0 +1,61 @@
+import type { TaskData } from "../Types";
+
+import { PipelineType } from "../../../widgets/src/lib/interfaces/Types";
+import { TASKS_MODEL_LIBRARIES } from "../const";
+
+const taskData: TaskData = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Benchmark dataset used for the task",
+			id:          "merve/coco2017",
+		}
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "image-segmentation-input.jpg",
+				type:     "img",
+			},
+		],
+		outputs: [
+			{
+				filename: "image-segmentation-output.jpg",
+				type:     "img",
+			},
+		],
+	},
+	id:        "image-segmentation",
+	label:     PipelineType["image-segmentation"],
+	libraries: TASKS_MODEL_LIBRARIES["image-segmentation"],
+	metrics:   [
+		{
+			description: "Average Precision (AP) is the Area Under the PR Curve (AUC-PR). It is calculated for each class separately.",
+			id:          "Average Precision",
```

Comment:
Nit: for this and other scripts, please keep id consistent. Use lower case, here it should be average_precision.